packages/l10n/src/constants.ts

```diff
@@ -0,0 +1,298 @@
+import type { timeZone_blocklist } from './types';
+
+export const TIMEZONE_BLOCKLIST: timeZone_blocklist = [
```

Comment:
I would omit the `timeZone_blocklist` and keep it like this

```suggestion
export const TIMEZONE_BLOCKLIST = [
```

PS: type names usually are in PascalCase, so `timeZone_blocklist` should have been `TimeZoneBlockList`.

Reply:
Also, can you provide a comment explaining what this blocklist is for? Thanks

Reply:
I think the type would be inferred. Also maybe naming it listOfUnsupportedTimeZones or excludedTimeZones might be clearer while adding a comment. 

@emmenko our idea was to not detach ourselves from moment's list entirely. As that also has drawbacks. As a result excluding felt more explicit and good common ground. I think a comment can help.

Reply:
Have you consider the other way around? So keeping an allow list only?

Reply:
Yes couldn't find much yet. The flow after this is intended to be if you pull/generate TZs that you are notified via a shell output to either accept the new TZ or add it to the deny list. As such we want to make denying a conscious choice not accepting a new TZ.

Reply:
I've updated the constant to be name `EXCLUDED_TIME_ZONES`, removed the type, and added a comment as to the intention behind this constant.

Let me know if we need to talk further about this implementation.